1.8/administration/installing/cloud/aws/advanced/template-reference.md

```diff
@@ -44,22 +44,22 @@ The Zen templates orchestrate the individual component templates.
     <th>Key Value</th>
   </tr>
   <tr>
-    <td>KeyName</td>
-    <td>AWS EC2 Key Pair to SSH into instances</td>
-  </tr>
-  <tr>
-    <td>AcceptEULA</td>
-    <td>Read the Mesosphere EULA and indicate agreement: https://docs.mesosphere.com/community-edition-eula/.</td>
+    <td>AdminLocation</td>
+    <td>Optional: Specify the IP range to whitelist for access to the admin zone. Must be a valid CIDR.</td>
```

Comment:
May also want to add to the description: To allow access from any IP address, use `0.0.0.0/0`